File: barcosEnContienda/src/gestorAplicacion/Items/Botin.java
package gestorAplicacion.Items;
import java.util.ArrayList;
//Clase cuya finalidad es ser transferible, esta compuesta por oro y los objetos que puede tener una isla o un barco
public class Botin {
	public int oro;
	public ArrayList<Objeto> objetos= new ArrayList<Objeto>();
	
	//Sobrecarga de constructores
	public Botin(int oro,ArrayList<Objeto> objetos) {
		this.oro=oro;
		this.objetos.addAll(objetos);
	}
	
	public Botin(int oro) {
		this.oro=oro;
	}
	
	public Botin(int oro, Objeto objeto) {
		this.oro=oro;
		this.objetos.add(objeto);
	}
}

File: practica-2-grupo-1-equipo-7-Barcos/src/gestorAplicacion/items/botin.py
class Botin:
    
    def __init__(self,oro=0, objetos=[]):
        self._objetos=[]
        self._oro=oro
        self._objetos.append(objetos)
    
    def getOro(self):
        return self._oro
    def getObjetos(self):
        return self._objetos